mtgjson5/set_builder.py

```diff
@@ -44,10 +44,26 @@
     UuidCacheProvider,
     WhatsInStandardProvider,
 )
+from .providers.enrichment_provider import EnrichmentProvider
```

Comment:
All provider should be imported into the `__init__` so they can be imported at top level, please